pkg/agent/const.go

```diff
@@ -96,6 +96,8 @@ const (
 	kubernetesDockerMonitorSystemdService     = "linux/cloud-init/artifacts/docker-monitor.service"
 	kubernetesContainerdMonitorSystemdTimer   = "linux/cloud-init/artifacts/containerd-monitor.timer"
 	kubernetesContainerdMonitorSystemdService = "linux/cloud-init/artifacts/containerd-monitor.service"
+	labelsScript                              = "linux/cloud-init/artifacts/labels.sh"
+	labelsSystemdService                      = "linux/cloud-init/artifacts/labels.service"
 	labelNodesScript                          = "linux/cloud-init/artifacts/label-nodes.sh"
 	labelNodesSystemdService                  = "linux/cloud-init/artifacts/label-nodes.service"
```

Comment:
these are not needed, can you help delete them? Thank you

Reply:
Happy to, but leaning towards another PR

Reply:
Deleted, and associated uses